### ISSUE TITLE
Unable to delete excluded workday day using keyboard

### ISSUE THREAD

**Issue by Nardol** [home-assistant/frontend#18003]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

By default, it looks like holidays are excluded from workday sensor. I only use keyboard (and screen reader) and I have not found at all how to delete an excluded day from the list. I thought I had to click on the button labelled by the day I want to delete but nothing happens.

### Describe the behavior you expected

I should be able to simply delete an excluded day using keyboard.

### Steps to reproduce the issue

1. Go to settings->Devices and integrations
2. Click work days
3. Click configure to set parameters for your workday sensor

### What version of Home Assistant Core has the issue?

2023.9.2

### In which browser are you experiencing the issue with?

Firefox 117.0.1 and Android Home Assistant companion app

### Which operating system are you using to run this browser?

Debian Buster, Android 13

### Additional information

As I understood, all days are represented by a button. I tried to click using space and a routine cursor from my braille display, nothing happens. Enter saved the settings and left click emulation of my screen reader had also no effect.
With this situation, i make some things which are done and cannot be undone.

**Comment by Nardol** [comment]
It is fixed, I have a button to delete the day for each button labeled by a day.
